src/main/java/com/zanox/internal/training/OuterAndInnerClass.java

```diff
@@ -28,8 +28,10 @@ private static void restrictedMethod() {
     private void otherRestricedMethod(){
         System.out.println("Accessed other private method!");
     }
+
     /**
      * Nested class to show creation from static out method
+     * - 'show creation from static out method' verstehe ich nicht
      * and as well to access a private static field of the outer class.
```

Comment:
Sorry supposed to be 'static method in outer class'
feel free to change

Reply:
Ok, I'll do